033-OSSDatabaseMigration/Coach/06-online-cutover-validation.md

```diff
@@ -0,0 +1,7 @@
+# Challenge 6: Online Cutover and Validation
+
+[< Previous Challenge](./05-online-migration.md) - **[Home](./README.md)** - [Next Challenge >](./07-private-endpoint.md)
+
+## Coach Tips
+
+1) It's a good idea to have the attendee register themselves as a new user so they can then go into psql/mysql/Azure Data Studio/MySQL WorkBench and check the users table to see that the data has indeed changed. Keep in mind that registering users will not work unless Google reCAPTCHA has been reconfigured. This is mentioned in the [Prerequisites](../Student/00-prereqs.md) 
```

Comment:
Same as last one, wrap the psql/mysql path in backticks so it shows monospaced.

Reply:
Fixed